### PR TITLE
Create udev rule to detect and symbolically link the IMU file descriptor to a file #6

### PR DESCRIPTION
**Pull request by ajarteag**
Removed "ros-noetic-imu-tools" from dockerfile to save time on docker compose up.
Will add back later in issue #9 if needed.

### CODE REVIEW

**Comment by ajarteag** [comment]
Used this [link as a resource](https://www.clearpathrobotics.com/assets/guides/kinetic/ros/Udev%20Rules.html)

**Comment by ajarteag** [comment]
This PR is NOT COMPLETE. Still need to get vendor and product IDs from the NGIMU.

File: docker-compose.yaml [hunk 1]
```diff
@@ -11,3 +11,4 @@ services:
     privileged: true
     volumes:
       - /dev:/dev
+      - /run/udev:/run/udev
```
Comment (matthewyjiang) [inline docker-compose.yaml:14]: Is this the correct directory for udev? i'm less experienced so i don't know everything but i believe udev rules live in /etc/udev but i could be wrong

File: docker-compose.yaml [hunk 2]
```diff
@@ -11,3 +11,4 @@ services:
     privileged: true
     volumes:
       - /dev:/dev
```
Comment (matthewyjiang) [inline docker-compose.yaml:13]: privileged: true

might already give /dev access. test this and remove the mount if it does.

**Review by matthewyjiang — CHANGES_REQUESTED** [review]

File: docker-compose.yaml (hunk 1 above)
Comment (ajarteag) [reply docker-compose.yaml:14]: i think u are correct, /etc/udev is where the udev rules files should live. 
however i think the /run/udev directory may serves a different purpose. will look more into it

**Pull request merged**